content/api/hub/sample_bts_golang.md

```diff
@@ -0,0 +1,21 @@
+title = "Behind the Scenes sample"
+template = "render_hub_content_body"
+date = "2024-01-15T00:22:56Z"
+content-type = "text/html"
+tags = ["golang", "web", "nuxt", "ssg"]
+
+[extra]
+author = "rajatjindal"
+type = "hub_document"
+category = "Sample"
+language = "Go"
+created_at = "2024-01-15T00:22:56Z"
+last_updated = "2024-01-15T00:22:56Z"
+spin_version = ">v1.5"
+summary =  "A fun social photo app built using Golang/Nuxt/Spin"
+url = "https://github.com/rajatjindal/behind-the-scenes"
+keywords = "slack, golang, nuxt.js, webhook"
```

Comment:
Feels like `slack` is interesting enough to merit a tag?  But not sure what keywords vs tags are for...

Reply:
keywords are not displayed as tags but help with search. Tags are shown on the card.

Reply:
I've added slack to tags